views/calendar/calendar.js

```diff
@@ -166,10 +166,9 @@ window.calendarView = class calendarView extends View {
             }
 
             function pushEvents(person, date, text, link, spouse) {
-                for (const monthName of MONTHS) {
```

Comment:
Ouch, there is probably one closing curly braces "}" that closed the for loop - please test it.

Reply:
Thanks @bcaseyrls